srcpkgs/smpq/patches/fix-smpq-compilation.patch

```diff
@@ -0,0 +1,52 @@
+--- a/CMakeLists.txt	2016-12-12 23:31:14.000000000 +0100
++++ b/CMakeLists.txt	2021-10-01 13:06:43.062603459 +0200
+@@ -18,6 +18,8 @@
+ #
+ 
+ project(SMPQ)
++file(GLOB_RECURSE CFILES "${CMAKE_SOURCE_DIR}/*.c")
++SET_SOURCE_FILES_PROPERTIES(${CFILES} PROPERTIES LANGUAGE CXX)
+ set(VERSION 1.6)
+ cmake_minimum_required(VERSION 2.6)
+ 
+@@ -43,9 +45,9 @@
+ 
+ 	try_compile(CHECK ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/check.c COMPILE_DEFINITIONS -I${STORMLIB_INCLUDE_DIR})
+ 
+-	if(NOT CHECK)
++	if(FALSE)
```

Comment:
Why not just remove the lines entirely?

Reply:
Using (FALSE), obtain the same result as you remove the rows. I refizes the patch by removing the lines.